closure/repositories.bzl

```diff
@@ -1097,9 +1097,9 @@ def rules_webtesting():
 
     http_archive(
         name = "io_bazel_rules_webtesting",
-        sha256 = "41d500a97ad9621dcf92fcb0cd77916e517388b196e5c3f0e63c7753e983b2bb",
```

Comment:
Sorry I don't understand its relation to runfile path?

Reply:
I really don't follow since you are not giving me enough context.

1. Are you saying currently the repository is broken and all webdriver tests are failing? 
2. If it is broken, how is J2CL CI not broken?
3. Why does this PR includes both? Do they have any relation?

Reply:
Could you pls revert the file?